agent/bench-scripts/test-bin/bm

```diff
@@ -5,6 +5,8 @@
 
 echo "$0 $*" >> $_testlog
 
+_exit_sts=${_BM_EXIT_STS:-0}
```

Comment:
This could have been collapsed into the `exit` command since `_BM_EXIT_STS` appears to be a McGuffin created exclusively for your new legacy test case, and `_exit_sts` isn't assigned anywhere else. I think that assigning it separately causes the eyes to try to follow all the intervening code to see where it might be changed... which turns out to be a waste of time. 😁